solidity/README.adoc

```diff
@@ -0,0 +1,51 @@
+:toc: macro
+
+= tBTC v2 contracts
+
+This package contains tBTC v2 contracts.
+
+toc::[]
+
+== Build, test and deploy
+
+tBTC v2 contracts use https://hardhat.org/[*Hardhat*] development environment.
+To build and deploy these contracts, please follow the instructions presented
+below.
+
+=== Prerequisites
+
+Please make sure you have the following prerequisites installed on your machine:
+
+- https://nodejs.org[Node.js] >12.22.1
```

Comment:
If we're transitioning to 14, maybe we should also require version >14 here, cause if we add a new dependecy that is actually required to have >14, then one can be surprised.

Reply:
https://github.com/keep-network/tbtc-v2/pull/17/commits/4d2620d4f93573dda27e8f30fa2811bc94ceacb4